### DISCUSSION TITLE
VSCode Compatibility

### DISCUSSION THREAD

**Discussion by danjebs** [community/community#9163] [Copilot Conversations]
I'm unable to install the VSCode extension - I'm receiving this error:

"Unable to install 'github.copilot' extension because it is not compatible with the current version of VS Code (version 1.46.0)."

![Screen Shot 2021-12-21 at 5 38 39 PM](https://user-images.githubusercontent.com/12756405/146907277-78f53b6c-4dd7-4f55-b61f-06be103be5c2.png)

Is this fixable on my end or should I play the waiting game?

**Comment by Urethramancer** [comment]
The latestst stable version is 1.63.2, while 1.46.0 is from June 2020. It might be so old it simply doesn't have all the interfaces the Copilot extension requires to function.
